Sources/SwiftTwitch/Classes/Results/Users/GetUsersFollowsData.swift

```diff
@@ -25,14 +25,4 @@ public struct GetUsersFollowsData: Unmarshaling {
     /// `Get Users Follows` call.
     public let paginationData: PaginationData?
 
-    /// Initializes a `GetUsersFollowsData` object from the input `MarshaledObject`. This will throw
-    /// if there is missing data from the input `MarshaledObject`.
-    ///
-    /// - Parameter object: The object to initialize a `GetUsersFollowsData` object from
-    /// - Throws: If data is missing that was expected to be non-`nil`.
-    public init(object: MarshaledObject) throws {
-        total = try object.value(for: Twitch.WebRequestKeys.total)
-        followData = try object.value(for: Twitch.WebRequestKeys.data)
```

Comment:
Different property name vs web request key name, need to change variable name or do custom Codable initialization